features/is-analytics-solution-features/org.wso2.carbon.sp.solutions.is.analytics.feature/src/test/resources/testng.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is there a need for testng.xml file inside features?

Reply:
No. Will remove it/